packages/htmlviewer-extension/src/index.tsx

```diff
@@ -108,19 +152,22 @@ function activateHTMLViewer(
   // allowing script executions in its context.
   app.commands.addCommand(CommandIDs.trustHTML, {
     label: trans.__('Trust HTML File'),
+    caption: trans.__(`Whether the HTML file is trusted.
+    Trusting the file allows scripts to run in it,
+    which may result in security risks.
+    Only enable for files you trust.`),
     isEnabled: () => !!tracker.currentWidget,
     isToggled: () => {
       const current = tracker.currentWidget;
       if (!current) {
         return false;
       }
-      const sandbox = current.content.sandbox;
-      return sandbox.indexOf('allow-scripts') !== -1;
```

Comment:
Not following this change. Actually not sure why it was there in the first place, but it might be better to make a separate PR for this?

Reply:
PR opened #10685